pkgs/applications/editors/hmm/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv
+, lib
+, makeWrapper
+, php
+, ncurses
+, coreutils
+, xclip
+}:
+
+
+stdenv.mkDerivation rec {
+  name = "h-m-m";
+
+  src = builtins.fetchGit {
+    url = "https://github.com/nadrad/h-m-m.git";
+    rev = "ecf74433844e5542f78f64f8a79807818743adff";
+  };
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -r h-m-m $out/bin/h-m-m
+  '';
+  postFixup = ''
+    wrapProgram $out/bin/h-m-m \
+      --set PATH ${lib.makeBinPath [
+        php
+        ncurses
+        coreutils
+        xclip
+      ]}
+  '';
+
+  meta = with lib; {
+    description = "h-m-m is a simple, fast, keyboard-centric terminal-based tool for working with mind maps.";
```

Comment:
This is not to be nit-picky, but it's the type of stuff people with commit access are gonna mention
> Don’t include a period at the end. Don’t include newline characters. Capitalise the first character. For brevity, don’t repeat the name of package — just describe what it does.
https://nixos.org/manual/nixpkgs/stable/#var-meta-description
```suggestion
    description = "A simple, fast, keyboard-centric terminal-based tool for working with mind maps";
```